ccd-definition/FixedLists/CareSupervision/dfj/SwanseaDFJCourts.json

```
@@ -0,0 +1,23 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "SwanseaDFJCourts",
+    "ListElementCode": "253",
+    "ListElement": "Llanelli",
+    "DisplayOrder": 1
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "SwanseaDFJCourts",
+    "ListElementCode": "274",
+    "ListElement": "Port Talbot ",
```

Extra space at the end
```suggestion
    "ListElement": "Port Talbot",
```

---

Removed extra space